SPECS/libvirt/libvirt.spec

```diff
@@ -18,6 +18,7 @@ URL:            https://libvirt.org/
 Source0:        https://libvirt.org/sources/%{name}-%{version}.tar.xz
 # CVE-2023-2700 is fixed by https://gitlab.com/libvirt/libvirt/-/commit/6425a311b8ad19d6f9c0b315bf1d722551ea3585
 Patch1:         CVE-2023-2700.patch
+Patch2:         CVE-2024-1441.patch
```

Comment:
I know NIST is currently not handing out CVE scores but this is graded as medium severity by RedHat. Why fast-track it?

Reply:
ACK, i actually saw it `HIGH` somewhere which is why i `fasttracked` it.
My bad ...

Reply:
No problem, thanks for the rebase!